robottelo/ui/puppetclasses.py

```diff
@@ -58,3 +58,60 @@ def delete(self, name, really):
 
         self.delete_entity(name, really, locators["puppetclass.select_name"],
                            locators['puppetclass.delete'])
+
+    def import_scap_client_puppet_classes(self):
+        """Imports puppet-foreman_scap_client puppet classes."""
+        Navigator(self.browser).go_to_puppet_classes()
+        self.wait_for_ajax()
+        self.find_element(locators['puppetclass.import']).click()
+        # Checking if the scap client puppet classes are already imported
+        if self.wait_until_element(
+                locators['puppetclass.environment_default_check'],
+                5,
```

Comment:
This method is supposed to return as soon as it finds the element. I think is not necessary to decrease the timeout, but @kbidarkar or @sghai can say this better.